packages/browser-tests/cypress/integration/console/schema.spec.js

```diff
@@ -18,6 +18,19 @@ describe("questdb schema with working tables", () => {
     });
     cy.refreshSchema();
   });
+  
+  it("should throw a warning when table is present with different case",()=>{
```

Comment:
Hey @AayushSaini101 , it looks to me that this test does not test your change.
This test goes to the backend to create the table.
What you would need to do is use cypress to open the table schema dialog, type in "Btc_Trades", and assert that the error message appeared.

I do not think you need to create any tables on the backend.
It looks to me that the 4 tables are created before every test in a `before()` hook, so they should exist already.
No need for this code:
```
    tables.forEach((table) => {
      cy.createTable(table);
    });
```

Check out this test in this file at line 43:
```
it("should filter the table with input field", () => {
  ...
}
```

Not exactly what you need, but it is a good example how to find elements on the UI with cypress, and how to simulate typing in text into an input, for example.

Reply:
Hope this works thanks @glasstiger 
```
  it("should throw a warning when table is present with different case",()=>{
    cy.get('input[name="table_filter"]').type("Btc_trades").should("contains","Table name must be unique")
    cy.get('input[name="table_filter"]').type("gitLog").should("contains","Table name must be unique")
    cy.get('input[name="table_filter"]').type("GITLOG").should("contains","Table name must be unique")
  })
```

Reply:
Hi @AayushSaini101 , is this test passing for you when you run it?

Reply:
Please, remove the test.
We can merge it.
It is a tiny change.

Reply:
Done @glasstiger Please merge